code/game/machinery/doors/airlock.dm

```diff
@@ -93,6 +93,7 @@ var/list/airlock_overlays = list()
 	hud_possible = list(DIAG_AIRLOCK_HUD)
 
 	var/air_tight = FALSE	//TRUE means density will be set as soon as the door begins to close
+	var/prying_so_hard = FALSE
```

Comment:
I pried so hard, and got so far, but in the end this fucking airlock won't open.